src/ui/Analytics/Analytics.ts

```diff
@@ -463,6 +469,14 @@ export class Analytics extends Component {
     );
   }
 
+  private handleCoveoUAEventReady(data: ICoveoUAEventArgs) {
+    if (this.options.populateDataLayer) {
+      let dataLayerName = this.options.dataLayerName;
```

Comment:
It's a best-practice to use `const` over `let` when the variable value does not change after initialization.

Reply:
Thanks, I'll remember this :)